### DISCUSSION TITLE
运行到网页后没有报错但是功能无法使用

### DISCUSSION THREAD

**Discussion by nighttosakura** [mikumifa/biliTickerBuy#81] [提问专区] [answered]
获取票信息和抢票按钮点了都没有反应，试了好几个版本都是同样的问题。

**Comment by mikumifa** [comment]
1. 别开梯子
2. 保证登陆成功 （登陆管理那里应该会显示自己的名字

如果不行的话就贴一个app.log

**Reply by nighttosakura** [reply, marked as answer]
最后发现原来是基础的权限问题，已解决。😇

**Reply by sbtv587** [reply]
> 1. 别开梯子
> 2. 保证登陆成功 （登陆管理那里应该会显示自己的名字
> 
> 如果不行的话就贴一个app.log

大佬，我打开之后，没有登录弹出来是什么原因啊
![微信图片_20240628123344](https://github.com/mikumifa/biliTickerBuy/assets/174082278/8f6c5504-8850-406a-aa6c-194a4effbea9)
